### ISSUE TITLE
Killcount 

### ISSUE THREAD

**Issue by PixelsOSRS** [pajlads/DinkPlugin#543]
### Checklist

- [X] I'm reporting a problem with Dink
- [X] I've searched the [issues and pull requests](https://github.com/pajlads/Dinkplugin/issues?q=) for similar looking reports

### Describe your issue

When you reach 1000 KC in a boss, Dink will send a screenshot to discord that it is de 1st kill you have done of this boss. When you put off Initial Boss Kill off there will be no more KC screenshots anymore. I set my Interval on 10 KC and when I reached 1010 KC, no screenshot was send either. I think it has something to do with how Jagex present KC in de chat, in the past KC above 1k where presented like 1000 and now it is presented as 1,000. Is there something that can fix this?

### Runelite Logs

[launcher_2024-09-11.0.log](https://github.com/user-attachments/files/16977305/launcher_2024-09-11.0.log)

### Runelite Version

Runelite Version: 1.10.37

**Comment by Chronoes** [comment]
Same here, I was about to post my issue report until I saw this

The "Kill Count" is reporting each kill as first kill, regardless of the KC. This may be specific to bosses or kill counts but I noticed it today when killing Kraken.

My Kraken KC was at 1117 and each of my kill since started posting to Discord as KC 1. Yesterday I was killing Vorkath and the plugin only reported my 200th KC as expected with 50 KC interval for posting,

![2024-09-12_12-53](https://github.com/user-attachments/assets/1e81484e-cd75-4eda-9db0-f9c4962ce0d8)
